### PR TITLE
Issue #198 guides navigation

### PR DESCRIPTION
**Pull request by flbulgarelli**
This PR adds previous and next arrows to the guide, similar to the nav arrows in the exercise.  

It is just a pure functional PR. It doesn't change layout nor style. That is another issue.

### CODE REVIEW

**Comment by faloi** [comment]
> @faloi Just before you ask:  to_s was added because ActiveAdmin sends that message in order to render object's as values. But if a `name` method is defined, it takes precedence. 

Seems much better that way. :smile_cat: 

PS: The next time, try to comment directly on the file and not in the commit, because GitHub doesn't provide reply functionality for that type of comments

File: app/models/concerns/with_path.rb [hunk 1]
```diff
@@ -9,4 +9,12 @@ module WithPath
   def next_guides
    path ? path.guides.where('guides.position > :position', position: position).order('guides.position asc') : []
   end
+
+  def next
+    path.guides.where(position: position + 1).first if path #FIXME duplicated in WithGuide
```
Comment (faloi) [inline app/models/concerns/with_path.rb]: Perhaps you can fix the FIXME :kissing_heart:

Comment (flbulgarelli) [inline app/models/concerns/with_path.rb]: Done

File: app/views/layouts/_guide.html.erb [hunk 2]
```diff
@@ -6,7 +6,16 @@
       <%= @guide.name %>
       <small> - <%= @guide.language %></small>
     </h1>
-    <h3><%= with_classifications  @guide %></h3>
+    <h4><%= with_classifications  @guide %></h4>
+
+    <% if @guide.path %>
+      <p>
+        <%= previous_nav_button(@guide) %>
```
Comment (faloi) [inline app/views/layouts/_guide.html.erb]: It does change the layout, can you provide a screenshot?

**Pull request merged**